composer/trainer/trainer.py

```diff
@@ -837,17 +861,9 @@ def __init__(
         if deterministic_mode:
             reproducibility.configure_deterministic_mode()
 
-        # Precision
-        if precision is None:
-            precision = Precision.AMP if isinstance(self._device, DeviceGPU) else Precision.FP32
-        if isinstance(precision, str):
-            precision = Precision(precision)
-
-        _validate_precision(precision, self._device, deepspeed_enabled)
-
         # Optimizers and Schedulers
         if not optimizers:
-            optimizers = DecoupledSGDW(list(model.parameters()), lr=0.1)
+            optimizers = DecoupledSGDW(model.parameters(), lr=0.1)
```

Comment:
why remove list?

Reply:
Oh this might have been just a nitpick as I was walking through the code, I don't think the list is necessary which is why I removed it? Let me try adding it back and see if anything breaks.

Reply:
Got it, yea I think it's fine to remove then. Just wanted to make sure it was intentional